src/Pos/Delegation/DB.hs

```diff
@@ -67,7 +67,7 @@ isIssuerByAddressHash :: MonadDBRead m => StakeholderId -> m Bool
 isIssuerByAddressHash = fmap isJust . getPskByIssuer . Right
 
 -- | Given issuer @i@ returns @d@ such that there exists @x1..xn@ with
--- @i->x1->...->xn->d@.
+-- @i→x1→...→xn→d@.
```

Comment:
Why do you sometimes remove non-ascii characters and sometimes add them? 😒 
WTF?